package.json

```diff
@@ -14,7 +14,7 @@
     "test": "yarn test:other && yarn test:js",
     "test:other": "yarn prettier --list-different",
     "test:js": "eslint --ignore-path .gitignore --ignore-path .prettierignore \"**/*.js\"",
-    "zip": "cd extension && jszip-cli add . -i .DS_Store -o ../extension.zip -f"
+    "zip": "jszip-cli add -i .DS_Store -o extension.zip -f dist/*"
```

Comment:
I'd like to either name this `dist.zip` or move it to `dist` to make the relationship with the other `dist` files more clear.

Reply:
Sure, I don't have a strong opinion about this! I think it could make sense to just output everything in `extension/` instead of `dist/`. I'm going to merge this now so we can unblock future PRs, but feel free to make a PR updating this.

Reply:
Ah, actually, I'll wait for your approval - didn't realize you requested changes. Please let me know what you think!

Reply:
Are you suggesting that we change `dist` to `extension`? I kinda like that idea since we'll won't need to change the unpackaged extension directory.

Reply:
That's right

Reply:
It's up to you if you want to merge this in its current state, I can rename the directory back in another PR

Reply:
Okay, sounds good. Let's merge this as is (since @lipis approved the current iteration), and we can always iterate going forward :)